zstacklib/zstacklib/utils/file_metadata_handler.py

```
@@ -0,0 +1,233 @@
+                )
+
+                summary_path = metadata_path + '.summary'
+                try:
+                    if os.path.isfile(summary_path):
+                        with open(summary_path, 'r') as sf:
+                            summary = json.loads(sf.read())
+                        entry.vmName = summary.get('vmName', '')
+                        entry.vmCategory = summary.get('vmCategory', '')
+                        entry.architecture = summary.get('architecture', '')
+                        entry.schemaVersion = summary.get('schemaVersion', '')
+                except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)